src/Shell/GettextShell.php

```diff
@@ -349,4 +348,33 @@ private function parseDir($dir) : void
             }
         }
     }
+
+    /**
+     * Extract translations from javascript files using ttag, if available.
+     *
+     * @return void
+     */
+    private function ttagExtract() : void
+    {
+        // check ttag and js folder exist
+        $ttag = 'node_modules/ttag-cli/bin/ttag';
+        $appDir = 'src/Template/Layout/js/app';
+        if (!file_exists($ttag) || !file_exists($appDir)) {
+            $this->out(sprintf('%s or %s not found', $ttag, $appDir));
```

Comment:
I'd change message a bit: `Skip javascript parsing - %s command not found` 
Reason: if `ttag` is used we should expect some js files to parse.